runtime/jvmti/jvmtiStartup.c

```diff
@@ -1110,6 +1281,98 @@ isAgentLibraryLoaded(J9JavaVM *vm, const char *library, BOOLEAN decorate)
 			j9mem_free_memory(agentPath);
 		}
 		Trc_JVMTI_isAgentLibraryLoaded_result(library, agentPath, result);
+#elif defined(AIXPPC) /* defined(LINUX) || defined(OSX) || defined(WIN32) */
+		/* The incoming library & decorate could be in the following cases:
+		 * - JvmtiAgent1 & TRUE
+		 * - libJvmtiAgent1.so & FALSE
+		 * - /path/to/libJvmtiAgent1.so & FALSE
+		 */
+		struct ld_info *objectFiles = NULL;
+		/* 32bit AIX needs 130 slots available for the base VM as of Java6sr7. This allows a little overhead
+		 * for 3rd party jni libraries. Adopt the same initial buffer size as
+		 * https://github.com/eclipse-openj9/openj9-omr/blob/openj9/port/aix/omrosbacktrace_impl.c#L176-L177
```

Comment:
These lines are too long, and the reference should refer to the function rather than line numbers; suggest:
```c
		/* Use the same initial buffer size as omrintrospect_backtrace_symbols_raw()
		 * in omr/port/aix/omrosbacktrace_impl.c.
		 */
```

Reply:
Updated